uint8 FIND_PATH=0
uint8 UNKNOWN_DESTINATION=1
uint8 SEARCH_FAILED=2

string goal_name
uint8 solver
---
uint8 result
string goal_floor
Node[] waypoints
